.gitignore

```diff
@@ -1 +1,79 @@
-/node_modules
```

Comment:
Joda, denne filen er generert. Men storparten her er da totalt unødvendig for oss. Er det ikke bedre å legge inn ting etterhvert som de kommer i veien?

Jeg hadde en slik global fil før, men har sluttet med det og legger nå bare inn enkelt linjer her og der.